Modules/System/Azure Blob Services API/src/ABSContainerContent.Table.al

```diff
@@ -49,23 +49,27 @@ table 9043 "ABS Container Content"
         field(12; "Last Modified"; DateTime)
         {
             DataClassification = SystemMetadata;
-            Caption = 'Last Modified';
+            Caption = 'Last-Modified';
+            Description = 'Caption matches the corresponding property as defined in https://docs.microsoft.com/en-us/rest/api/storageservices/list-blobs#response-body';
         }
         field(13; "Content Length"; Integer)
         {
             DataClassification = SystemMetadata;
-            Caption = 'Content Length';
+            Caption = 'Content-Length';
```

Comment:
Shouldn't the Captions beeing set to `Locked = true` ?

Reply:
Very true :) Thanks!